config/src/lib.rs

```diff
@@ -268,6 +280,15 @@ pub struct RelayerConfig {
 
     /// Whether or not to enable Datadog-formatted logs
     pub datadog_enabled: bool,
+
+    /// Whether or not to enable metrics collection
+    pub metrics_enabled: bool,
+
+    /// The StatsD recorder host to send metrics to
+    pub statsd_host: String,
+
+    /// The StatsD recorder port to send metrics to
+    pub statsd_port: u16,
```

Comment:
nit: maybe less white space, following the rest of the struct? I understand this to be a stylistic difference, but I'd suggest maintaining convention when one is established

Reply:
oh ya i agree, i think maybe my formatter config did this